src/services/measure.service.js

```diff
@@ -278,15 +278,15 @@ const evaluateMeasureForPopulation = async (args, { req }) => {
     });
 
     logger.info('Successfully generated $evaluate report');
-    return results;
+    return wrapReportsInBundlesParameters(results);
```

Comment:
```suggestion
    return wrapReportsInBundlesParameters([results]);
```
The summary report for a population has a single MeasureReport result rather than an array of MeasureReport.